File: week2/evanotero/FizzBuzz.java
public class FizzBuzz {
    static public void main(String args[]) {
        for (int num = 1; num <= 100; num++) {
            String s = "";
            s += num % 3 == 0 ? "Fizz" : "";
            s += num % 5 == 0 ? "Buzz" : "";
            System.out.println(!s.isEmpty() ? s : num);
        }
    }
}

File: week2/evanotero/fizzbuzz.py
# FizzBuzz

def fizzbuzz(n):
    s = ""
    if n % 3 == 0:
        s += "Fizz"
    if n % 5 == 0:
        s += "Buzz"
    return s if s else n

for num in range(1,101):
    print fizzbuzz(num)
